.github/workflows/upstream-integration-test.yml

```diff
@@ -44,6 +44,8 @@ on:
           - 'sgp3'
           - 'aus3'
 
+run-name: ${{ inputs.id }}
```

Comment:
What happens when there is no id passed? Since id is optional parameter.

Reply:
Looks like we can use github.job to get the job id assigned by github: https://docs.github.com/en/actions/learn-github-actions/contexts#github-context